services/QuillLMS/db/migrate/20241121214425_add_feedback_history_index_on_several_columns.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddFeedbackHistoryIndexOnSeveralColumns < ActiveRecord::Migration[7.1]
+  # An index can be created concurrently only outside of a transaction.
+  disable_ddl_transaction!
+
+  def up
+    execute <<~SQL
+      CREATE INDEX CONCURRENTLY index_feedback_histories_on_multi_columns
+      ON feedback_histories(rule_uid, prompt_id, activity_version, used)
+      INCLUDE (id, metadata);
+    SQL
+  end
+
+  def down
+    execute <<~SQL
+      DROP INDEX IF EXISTS index_feedback_histories_on_multi_columns;
```

Comment:
Do we need `CONCURRENTLY` on the `DROP`, too?

Reply:
I'm thinking 'no', since the DROP INDEX command is quite fast